core/controllers/resources.py

```diff
@@ -52,7 +52,7 @@ class AssetDevHandler(base.BaseHandler):
     image and audio files are served from GCS).
     """
 
-    _SUPPORTED_TYPES = ['image', 'audio']
+    _SUPPORTED_TYPES = ['image', 'audio', 'thumbnails']
```

Comment:
Should it be "thumbnail" to follow the other nouns which are singular?

Reply:
Done.